src/electionguard/decrypt_with_shares.py

```diff
@@ -25,6 +30,8 @@ def decrypt_tally(
     tally: CiphertextTally,
     shares: Dict[GuardianId, DecryptionShare],
     crypto_extended_base_hash: ElementModQ,
+    manifest: Manifest,
+    remove_placeholders: bool = True,
```

Comment:
This could be very important for testing purposes. We are not using it now, but I would argue that it is already in place in `decrypt_with_secrets` and it should be kept.

https://github.com/microsoft/electionguard-python/blob/caa4d7553b36797aeb72692028b3040754444ad6/src/electionguard/decrypt_with_secrets.py#L124

Reply:
This method is also the base method. The decryption mediator is NOT the top level. This is. The decryption mediator at some point should be moved to a stateful class.

Reply:
I'm a big fan of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it), but I don't feel that strongly about it in this instance, so I'll concede the point